packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -300,7 +311,9 @@ class Shortcuts extends StatefulWidget {
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
     properties.add(DiagnosticsProperty<ShortcutManager>('manager', manager));
-    properties.add(DiagnosticsProperty<Map<LogicalKeySet, Intent>>('shortcuts', shortcuts));
+    final bool showDebugLabel = debugLabel != null && debugLabel.isNotEmpty;
+    properties.add(StringProperty('debugLabel', debugLabel, defaultValue: null, level: showDebugLabel ? DiagnosticLevel.info : DiagnosticLevel.hidden));
+    properties.add(DiagnosticsProperty<Map<LogicalKeySet, Intent>>('shortcuts', shortcuts, level: showDebugLabel ? DiagnosticLevel.fine : DiagnosticLevel.info));
```

Comment:
Rather than using two different properties one for debugLabel and one for the shortcuts,
consider instead adding specifying the 
description parameter in the 'shortcuts' property indicating it is its default value instead of the overly long value from the toString.

Reply:
I'm not sure I understand quite what you mean.  Like this?
```dart
    final bool showDebugLabel = debugLabel?.isNotEmpty ?? false;
    properties.add(
        DiagnosticsProperty<Map<LogicalKeySet, Intent>>(
            'shortcuts',
            shortcuts, description: showDebugLabel ? debugLabel : null,
            level: showDebugLabel ? DiagnosticLevel.info : DiagnosticLevel.fine),
    );
```

Reply:
yep!